calc.sh

```diff
@@ -13,10 +13,10 @@ echo "Enter your choice"
 read ch
 
 case $ch in
-1)sum=`expr $n1 + $n2`
+1)sum=`expr $n1 - $n2`
    echo "Sum ="$sum;;
 
-2)sum=`expr $n1 + $n2`
+2)sum=`expr $n1 - $n2`
```

Comment:
Can you pls give a comment inwhich line you are fixing bug

Reply:
Added a comments as well.